ocp/cosa-image.yaml

```diff
@@ -0,0 +1,220 @@
+        local: true
+      tags:
+      - from:
+          kind: DockerImage
+          name: 'registry.fedoraproject.org/fedora:32'
+        name: '32'
+        referencePolicy:
+          type: Source
+        importPolicy:
+          scheduled: true
+
+  ### COREOS-ASSEMBLER Local Image Stream ###
+  # This is the image stream that local builds push to
+  - apiVersion: image.openshift.io/v1
+    kind: ImageStream
+    metadata:
+      name: coreos-assembler
+    spec:
+      lookupPolicy:
+        local: true
+      failedBuildHistoryLimit: 1
+      successfulBuildsHistoryLimit: 1
+      lookupPolicy:
+        local: true
+
+  ### COREOS-ASSEMBLER Import Stream
+  - apiVersion: image.openshift.io/v1
+    kind: ImageStream
+    metadata:
+      name: coreos-assembler-import
+    spec:
+      lookupPolicy:
+        local: true
+      failedBuildHistoryLimit: 1
+      successfulBuildsHistoryLimit: 1
+      lookupPolicy:
+        local: true
+      tags:
+        # import the `master` image just in-case
+        - name: master
+          from:
+            kind: DockerImage
+            name: quay.io/coreos-assembler/coreos-assembler:master
+          importPolicy:
+            scheduled: true
+        # import the 4.3.x build master
+        - name: rhcos-4.3
+          from:
+            kind: DockerImage
+            name: quay.io/coreos-assembler/coreos-assembler:rhcos-4.3
+          importPolicy:
+            scheduled: true
+        # import the 4.4.x build master
+        - name: rhcos-4.4
+          from:
+            kind: DockerImage
+            name: quay.io/coreos-assembler/coreos-assembler:rhcos-4.4
+          importPolicy:
+            scheduled: true
+        # import the 4.5.x build master
+        - name: rhcos-4.5
+          from:
+            kind: DockerImage
+            name: quay.io/coreos-assembler/coreos-assembler:rhcos-4.5
+          importPolicy:
+            scheduled: true
+
+  ### Master Build
+  - apiVersion: build.openshift.io/v1
+    kind: BuildConfig
+    metadata:
+      labels:
+        app: ${IMAGE}
+      name: ${IMAGE}-master
+    spec:
+      failedBuildHistoryLimit: 1
+      successfulBuildsHistoryLimit: 1
+      # kill the build after 3hrs
+      completionDeadlineSeconds: 13200
+      lookupPolicy:
+        local: true
+      output:
+        to:
+          kind: ImageStreamTag
+          name: ${IMAGE}:master
+      postCommit: {}
+      resources:
+        limits:
+          memory: 4000Mi
+      runPolicy: Serial
+      source:
+        git:
+          ref: "${REPO_REF}"
+          uri: "${REPO_URL}"
+        type: Git
+      strategy:
+        dockerStrategy:
+          noCache: true
+          env:
+              - name: GIT_URL
+                value: "${REPO_URL}"
+              - name: GIT_REF
+                value: "${REPO_REF}"
+              - name: OPENSHIFT_GIT_HACK
+                value: "TRUE"
+          from:
+            kind: ImageStreamTag
+            name: fedora:32
```

Comment:
We should probably template these, today they are traps for whenever we want to rebase cosa.  I think we may be able to tell openshift builds to just honor what the `Dockerfile` has in `FROM`?